models/twilio_number_overview.sql

```diff
@@ -1,15 +1,15 @@
-with message_enhanced as (
+with message_enhanced as ( 
 
     select *
     from {{ ref('twilio_message_enhanced')}}
 )
 
 select
-    phone_number,
-    count(case
-        when direction like '%outbound%'
-        then message_id end)
-        as total_outbound_messages,
+    phone_number, 
+    count(case 
+        when direction like '%outbound%' 
+        then message_id end) 
+        as total_outbound_messages, 
```

Comment:
I imagine we can make this significantly dryer by making this into a for loop. Can you rework to be a for loop.

Reply:
Good point, changed to using a for loop